src/middleware/devtools.ts

```diff
@@ -88,8 +88,9 @@ export const devtools =
         : options
 
     const extensionConnector =
-      (window as any).__REDUX_DEVTOOLS_EXTENSION__ ??
-      (window as any).top.__REDUX_DEVTOOLS_EXTENSION__
+      typeof window !== 'undefined' &&
```

Comment:
Maybe it is also an idea to add an early return in the very beginning (line 83) of the devtools function?
`if (typeof window === 'undefined') return fn(set, get, api)`
It would also make it more obvious that the devtools are only running in a browser environment.

That way you would not have to deal with these window checks throughout the function.

Reply:
Your changes resolved the issue.
Thank you for reacting that fast. This is not a matter of course 🙏🏼

Reply:
good point. hm, but then we need two returns. let's try...

Reply:
Looks better to me 👍🏻